lib/geom/int_position.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flame/position.dart';
```

Comment:
Why do we need a Position class of Int? Is this just a copy of Position, but using integers?

Reply:
yup, that is exactly it. for integer handling? hahaha using doubles in place of integers my lead to precision issues, I guess. also makes the apis nicer when using, avoiding casts. We use that a lot on tales, but I think often of getting rid of it. I don't have a strong opinion about it yet.